templates/pages/management.html

```diff
@@ -0,0 +1,227 @@
+{% extends '_base.html' %}
+{% load static %}
+
```

Comment:
The page looks great! We need to break it down into functional components though. It's going to be very tough to write this page if all the functionality is going to be in one template.